protos/gossip/message.proto

```diff
@@ -130,6 +130,12 @@ message GossipMessage {
         // Encapsulates private data used to distribute
         // private rwset after the endorsement
         PrivateDataMessage private_data = 25;
+
+        // Contains validation results
+        ValidationResultsMessage validation_results_msg = 50;
+
+        // Request for block validaton
+        DataMessage validation_req_msg = 51;
```

Comment:
Same - should not be part of this commit.

Reply:
done